### PR TITLE
Update cleaner bot guides

### PR DESCRIPTION
**Pull request by espendk**

### CODE REVIEW

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+**Prerequisites**
+
+- `taker` must have approved Mangrove for spending `inbound_tkn` on the offer list you want to clean.
+- `taker` must have sufficient `inbound_tkn` funds for the execute arguments
+
+**Steps**
+
+1. Call `Mangrove.cleanByImpersonation(olKey, targets, taker)` specifying the offer list, an array of offers to clean, and the taker to impersonate.
+    - The targert are `MgvLib.CleanTarget` structs which has the following fields:
+        - `offerId`: The offer to clean
```
Comment (dontrolle) [inline docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md:28]: Nitpick: End `offerId` bullet-point with a full stop (like the other points).

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+The return values are the number of successfully cleaned offers and the total bounty received.
+
+Note that Mangrove won't attempt to execute an offer if the values in a `CleanTarget` don't match its offer (as desribed above). To distinguish between a non-executed clean and a fail clean (due to the offer itself not failing), you must inspect the log (see `MgvLib.sol`) or check the received bounty.
+
+
+## Example
+
+You have identified a whale that has 2,000,000 DAI and has approved Mangrove for spending its DAI.
+
+You're running a cleaning bot and it has detected that offer #708 on the (WETH, DAI, 1) offer list will fail when fully taken. The offer `gives` 615.3842 WETH at tick 75,171, corresponding to a price of 1,838.53 DAI/WETH. Thus, fully taking it requires 1,131,396.159384 DAI which corresponds to a raw value of `113139615938400000000000000`.
```
Comment (dontrolle) [inline docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md:45]: "[...] on the (WETH, DAI, 1) offer list [...]" - maybe add a friendly link to the offer list explainer page, for the reader that don't have it in short-term memory that the third parameter is tick-spacing?

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+1. Call `Mangrove.cleanByImpersonation(olKey, targets, taker)` specifying the offer list, an array of offers to clean, and the taker to impersonate.
+    - The targert are `MgvLib.CleanTarget` structs which has the following fields:
+        - `offerId`: The offer to clean
+        - `tick`: The offer's expected tick. If this doesn't match when the tx is executed, cleaning of the offer will not be attempted to avoid wasting gas since the cleaning assumptions have changed.
+        - `gasreq`: The offer's expected gasreq. If this is lower than the offer's actual gasreq when the tx is executed, cleaning of the offer will not be attempted to avoid wasting gas since the cleaning assumptions have changed.
+        - `takerWants`: The amount to request from the offer.
+2. The collected bounties (in native token) will be transferred to `msg.sender`.
+
+If an offer doesn't fail, only the execution of that offer will be reverted; The call to `cleanByImpersonation` will not revert.
```
Comment (dontrolle) [inline docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md:34]: Nitpick: Full stop between the sentences, or don't capitalize "; The (call) ..."

**Review by dontrolle — COMMENTED** [review]
Although not part of this changeset, `/developers/keeper-bots/background/the-role-of-cleaning-bots-in-mangrove` should be updated also. Whether it be part of this changeset or a follow-up is less important.

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md (hunk 2 above)
Comment (espendk) [reply docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md:45]: I'll remove the tick spacing, it's not important here.

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md (hunk 3 above)
Comment (espendk) [reply docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md]: Addressed in [e7881cd](https://github.com/mangrovedao/mangrove-docs/pull/204/commits/e7881cda67178bbc9ad90f64f55d9bd3fbda3c6a)

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md (hunk 2 above)
Comment (espendk) [reply docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md]: Addressed in [e7881cd](https://github.com/mangrovedao/mangrove-docs/pull/204/commits/e7881cda67178bbc9ad90f64f55d9bd3fbda3c6a)

File: docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md (hunk 1 above)
Comment (espendk) [reply docs/developers/keeper-bots/guides/use-borrowed-funds-for-cleaning.md]: Addressed in [e7881cd](https://github.com/mangrovedao/mangrove-docs/pull/204/commits/e7881cda67178bbc9ad90f64f55d9bd3fbda3c6a)

**Comment by espendk** [comment]
> Although not part of this changeset, `/developers/keeper-bots/background/the-role-of-cleaning-bots-in-mangrove` should be updated also. Whether it be part of this changeset or a follow-up is less important.

Good catch. Addressed in [e7881cd](https://github.com/mangrovedao/mangrove-docs/pull/204/commits/e7881cda67178bbc9ad90f64f55d9bd3fbda3c6a)

**Review by dontrolle — APPROVED** [review]
good to go

**Pull request merged**